index.rst

```diff
@@ -108,6 +107,7 @@ the ``GodotEngine.epub`` file in an e-book reader application.
    :name: sec-tutorials
 
    tutorials/best_practices/index
+   tutorials/troubleshooting
```

Comment:
A bit worried about this part because I don't want to mess up the translations.
Since the translation lookup happens via paths and the `troubleshooting.rst` is now in a different location, I don't know whether the `troubleshooting` section translation will be lost or set to fuzzy after the merge.

Hopefully the process recognises a simple move without changes, but better be safe